volatility/framework/automagic/mac.py

```diff
@@ -118,6 +118,66 @@ def stack(cls,
 class MacUtilities(object):
     """Class with multiple useful mac functions."""
 
+    @classmethod
+    def mask_mods_list(cls,
```

Comment:
Are `Module` objects what we're doing here, or is it a specific type?  If so, would it be useful code for all `Module` objects?  It feels like it's related to [the `mask_symbol_table` function](https://github.com/volatilityfoundation/volatility3/blob/master/volatility/framework/symbols/__init__.py#L248) in some way?

Reply:
It is a list of linux kernel modules gathered by walking the list. To do a proper symbol lookup we have to mask the addresses to mask how Volatility treats them.

Reply:
Ok, cool.  I'm happy to commit it you are?